2.1/runtime/nanoserver-1709/amd64/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# escape=`
+
+# Installer image
+FROM microsoft/windowsservercore:1709 AS installer-env
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Retrieve .NET Core Runtime
+ENV DOTNET_VERSION 2.1.0-preview1-25816-02
+ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-$DOTNET_VERSION-win-x64.zip
+ENV DOTNET_DOWNLOAD_SHA 1F7BB23909CBBEA1A010773FD9E7BFE9944E551FEB7DCF10AC5D923901BD13BEB09A2677D528F16398329FD892B4B2975404D842581475C703633C99DD069CC5
+
+RUN Invoke-WebRequest $Env:DOTNET_DOWNLOAD_URL -OutFile dotnet.zip; `
```

Comment:
Suggest using `UseBasicParsing` parameter for `Invoke-WebRequest` cmdlet

Reply:
I logged this as a separate issue as I think it should be considered and applied to all of the Windows based Dockerfiles not just the ones I am adding here - https://github.com/dotnet/dotnet-docker-nightly/issues/469